packages/plugin-server-session/test/session.test.js

```diff
@@ -140,4 +140,27 @@ describe('plugin: server sessions', () => {
     expect(sessionClient.breadcrumbs.length).toBe(2)
     expect(Object.keys(sessionClient._metadata).length).toBe(2)
   })
+
+  it('should support pausing/resuming sessions', () => {
```

Comment:
Some inline comments in these tests would be useful to clarify expected behaviour and what is being asserted.